bench/php/optimized/hash_xxhash.ll

```diff
@@ -2576,23 +2576,24 @@ XXH3_len_17to128_64b.exit:                        ; preds = %113, %183
   br i1 %exitcond.not.i, label %267, label %250
 
 267:                                              ; preds = %250
-  %268 = trunc i64 %1 to i32
-  %269 = lshr i64 %266, 37
-  %270 = xor i64 %269, %266
-  %271 = mul i64 %270, 1609587791953885689
-  %272 = lshr i64 %271, 32
-  %273 = xor i64 %272, %271
-  %274 = icmp ugt i32 %268, 143
-  br i1 %274, label %.lr.ph.preheader.i, label %XXH3_len_129to240_64b.exit
+  %268 = lshr i64 %266, 37
+  %269 = xor i64 %268, %266
+  %270 = mul i64 %269, 1609587791953885689
+  %271 = lshr i64 %270, 32
+  %272 = xor i64 %271, %270
+  %273 = icmp ugt i64 %1, 143
+  br i1 %273, label %.lr.ph.preheader.i, label %XXH3_len_129to240_64b.exit
 
 .lr.ph.preheader.i:                               ; preds = %267
-  %275 = lshr i64 %1, 4
-  %wide.trip.count.i = and i64 %275, 268435455
+  %274 = trunc i64 %1 to i32
+  %275 = lshr i32 %274, 4
+  %umax.i = tail call i32 @llvm.umax.i32(i32 %275, i32 9)
+  %wide.trip.count.i = zext nneg i32 %umax.i to i64
```

Comment:
Regression.

Reply:
Duplicate of https://github.com/dtcxzyw/llvm-opt-benchmark/pull/600#discussion_r1597406008